scripts/var_props.py

```diff
@@ -938,7 +942,6 @@ def __init__(self, var1_stdname, var1_type, var1_kind, var1_units,
         if self.__compat:
             # Check units argument
             if var1_units != var2_units:
-                self.__equiv = False
```

Comment:
Is this intentional?

Reply:
Yeah. It's a bug. Units can be equivalent if a transform exists.

Reply:
I thought the existence of a transform means the variables are "compatible" rather than "equivalent"?

(sorry, confused!)